example/android/shared/src/commonMain/kotlin/com/myrnproject/shared/MyComposeView.kt

```diff
@@ -0,0 +1,46 @@
+package com.myrnproject.shared
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.rememberCoroutineScope
+import de.voize.reaktnativetoolkit.annotation.ReactNativeProp
+import de.voize.reaktnativetoolkit.annotation.ReactNativeViewManager
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+
+@Composable
+@ReactNativeViewManager("MyComposeView")
+internal fun MyComposeView(
+    @ReactNativeProp
+    message: Flow<String>,
+    @ReactNativeProp
+    textFieldValue: Flow<String>,
+    @ReactNativeProp
+    onTextFieldValueChange: (String) -> Unit,
+    @ReactNativeProp
+    onButtonPress: () -> Unit,
+    // dependency injection
+    persistentConfig: PersistentConfig,
```

Comment:
That's nice

Reply:
I reverted to only allowing Kotlin dependencies and will do the react native props in a separate MR